app/points/domain/case-type-weightings.js

```diff
@@ -1,10 +1,11 @@
 const assertNumber = require('./validation/assert-number')
 
 class CaseTypeWeightings {
-  constructor (warrants, unpaidWork, overdueTermination) {
+  constructor (warrants, unpaidWork, overdueTermination, commTierCPWeighting = 0) {
     this.warrants = warrants
     this.unpaidWork = unpaidWork
     this.overdueTermination = overdueTermination
+    this.commTierCPWeighting = commTierCPWeighting
```

Comment:
I don't think we need this in here as if it's not enabled we just won't call it and if it is enabled it will be always be 1.